app/src/main/res/layout/row_suggestion.xml

```diff
@@ -8,5 +8,7 @@
     android:gravity="center_vertical"
     android:minHeight="?attr/listPreferredItemHeightSmall"
     android:padding="8dp"
-    android:textAppearance="?attr/textAppearanceSearchResultTitle"
+    android:textAppearance="?android:attr/textAppearanceMedium"
+    android:textColor="?android:textColorPrimary"
```

Comment:
Setting manually text color was needed otherwise text would have been gray instead of black. I had no idea why.

Reply:
The `textAppearanceMedium` style probably sets secondary text color.